ElementX/Sources/Screens/RoomDetailsScreen/RoomDetailsScreenModels.swift

```diff
@@ -261,3 +265,26 @@ enum RoomDetailsScreenErrorType: Hashable {
     /// Leaving room has failed..
     case unknown
 }
+
+enum PinnedEventsActionState {
```

Comment:
I know its long-winded, but this should probably be namespaced to the screen somehow or put in a shared location.
```suggestion
enum RoomDetailsScreenPinnedEventsActionState {
```